utilities/blob_db/blob_db_test.cc

```diff
@@ -61,40 +66,59 @@ class BlobDBTest : public testing::Test {
     myoptions.create_if_missing = true;
     EXPECT_TRUE(
         BlobDB::Open(myoptions, bblobdb_options, dbname_, &blobdb_).ok());
+    ASSERT_NE(nullptr, blobdb_);
   }
 
-  void insert_blobs() {
-    WriteOptions wo;
-    ReadOptions ro;
-    std::string value;
-
-    ColumnFamilyHandle *dcfh = blobdb_->DefaultColumnFamily();
-
-    Random rnd(301);
-    for (size_t i = 0; i < 100000; i++) {
-      int len = rnd.Next() % 16384;
-      if (!len) continue;
+  void PutRandomWithTTL(const std::string &key, int32_t ttl, Random *rnd,
+                        std::map<std::string, std::string> *data = nullptr) {
+    int len = rnd->Next() % 16384 + 1;
```

Comment:
16384 is 2^16. (2 >> 16) would be much more readable here imo instead of 16384; the reason being, the first time i saw this line, I was left wondering what this magic number is for a brief moment ... and other people might have the same thought.